memberships/urls.py

```diff
@@ -1,9 +1,20 @@
 from django.urls import path
 from . import views
+from django.contrib.auth.views import *
 
 urlpatterns = [
+    # path("/", views.home, name="memberships_home"),
```

Comment:
Could we remove commented out code please :)

Reply:
Indubitably!